storage/tianmu/types/value_parser4txt.h

```diff
@@ -59,22 +59,22 @@ class ValueParserForText {
     return nullptr;
   }
 
-  static common::ErrorCode ParseNumeric(BString const &rcs, int64_t &out, common::CT at);
-  static common::ErrorCode ParseBigIntAdapter(const BString &rcs, int64_t &out);
-  static common::ErrorCode ParseDecimal(BString const &rcs, int64_t &out, short precision, short scale);
-  static common::ErrorCode ParseDateTimeAdapter(BString const &rcs, int64_t &out, common::CT at);
+  static common::ErrorCode ParseNumeric(BString const &tianmu_s, int64_t &out, common::CT at);
+  static common::ErrorCode ParseBigIntAdapter(const BString &tianmu_s, int64_t &out);
+  static common::ErrorCode ParseDecimal(BString const &tianmu_s, int64_t &out, short precision, short scale);
+  static common::ErrorCode ParseDateTimeAdapter(BString const &tianmu_s, int64_t &out, common::CT at);
 
-  static common::ErrorCode Parse(const BString &rcs, RCNum &rcn, common::CT at);
-  static common::ErrorCode ParseNum(const BString &rcs, RCNum &rcn, short scale);
-  static common::ErrorCode ParseBigInt(const BString &rcs, RCNum &out);
-  static common::ErrorCode ParseReal(const BString &rcbs, RCNum &rcn, common::CT at);
-  static common::ErrorCode ParseDateTime(const BString &rcs, RCDateTime &rcv, common::CT at);
+  static common::ErrorCode Parse(const BString &tianmu_s, TianmuNum &tianmu_n, common::CT at);
+  static common::ErrorCode ParseNum(const BString &tianmu_s, TianmuNum &tianmu_n, short scale);
+  static common::ErrorCode ParseBigInt(const BString &tianmu_s, TianmuNum &out);
+  static common::ErrorCode ParseReal(const BString &tianmu_bs, TianmuNum &tianmu_n, common::CT at);
```

Comment:
check the lines above or below this line , the first argument is named tianmu_s, only this is nameed tianmu_bs. Better to uniform it.